entrypoint.sh

```diff
@@ -16,12 +16,14 @@
 
 OPENSDS_HOTPOT_URL=${OPENSDS_HOTPOT_URL:-http://127.0.0.1:50040}
 OPENSDS_ORCHESTRATION_URL=${OPENSDS_ORCHESTRATION_URL:-http://127.0.0.1:5000}
+SODA_DELFIN_URL=${SODA_DELFIN_URL:-http://127.0.0.1:8190}
```

Comment:
Is this a configurable (user given ) setting but defaults to 8190 ?

Reply:
This is the configuration file for the installation. If the Port is going to be dynamic then it will have to be set in the installation steps and read here during the setup.